gui/compsel/comp_explore.m

```diff
@@ -671,12 +671,12 @@ function refresh_comp_explore(h, varargin)
     % erpimage deletes current axis :( - we must use its altered version
     % remember about 'filt'  = [low_boundary high_boundary] option
     if ~h.opt.tri.filt || length(h.spct.freqsel) < 2
-        [~,~,~,~,h.triaxhndls] = erpimage2( h.EEG.icaact(h.comp,:,:) ...
+        [~,~,~,~,h.triaxhndls] = erpimage_silent( h.EEG.icaact(h.comp,:,:) ...
```

Comment:
BTW: jeżeli chemy być backwards compatible jak tylko się da (nie będzie to proste), to można rozważyć stosowanie zamiast `~` jakichś tymczasowych zmiennych np. `tmp1`. Wiele zależy od tego w której wersji wprowadzono `~` - jeżeli stosunkowo niedawno (powiedzmy max 6 lat temu), to lepiej będzie ich unikać. :disappointed: